src/top_model/main.cpp

```diff
@@ -209,5 +214,12 @@ std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_share
     r.run_until(NDTime("04:00:00:000"));
     auto elapsed = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
     cout << "Simulation took:" << elapsed << "sec" << endl;
+	
+	FILE *input;
+	FILE *output;
+	input = fopen("../data/abp_output.txt","r");
```

Comment:
Kindly change the paths, they are incorrect. New paths will be,
`input = fopen("./data/abp_output.txt","r");`
`output = fopen("./data/abp_processed_output.csv","w");`

Reply:
After discussion, we think that there is an issue related to file paths. For me, the suggested file paths work fine. Whereas, for the doer's their implemented file path create the correct output. Hence, this can request be bypassed.